lib/src/base.dart

```diff
@@ -201,10 +201,12 @@ abstract class SentryClient {
   Future<SentryResponse> captureException({
     @required dynamic exception,
     dynamic stackTrace,
+    dynamic extra,
   }) {
     final event = Event(
       exception: exception,
       stackTrace: stackTrace,
+      extra: extra
```

Comment:
nit: add a trailing comma (this will tell `dartfmt` to put each argument on its own line)